client/Windows/wf_event.c

```diff
@@ -746,7 +745,7 @@ BOOL wf_scale_blt(wfContext* wfc, HDC hdc, int x, int y, int w, int h, HDC hdcSr
 	if (!wh)
 		wh = dh;
 
-	if (wfc->fullscreen || !wfc->context.settings->SmartSizing || (ww == dw && wh == dh))
+	if (wfc->fullscreen || !wfc->common.context.settings->SmartSizing || (ww == dw && wh == dh))
```

Comment:
couldn't it be`if (wfc->fullscreen || !settings->SmartSizing || (ww == dw && wh == dh))` ?